docs/tutorials/qemu-remote-tutorial.md

```diff
@@ -0,0 +1,123 @@
+# Podman Remote clients for macOS and Windows with QEMU
+
+***
+**_NOTE:_** For running Podman on Windows, refer to the [Podman for Windows](podman-for-windows.md) guide, which uses the recommended approach of a Podman-managed Linux backend. For Mac, see the [Podman installation instructions](https://podman.io/getting-started/installation). This guide covers the advanced usage of Podman with a custom Linux VM or a remote external Linux system.
+***
+
+## Introduction
+
+There are officially supported podman machines with QEMU on macOS and WSL2 on Windows, but if one wants to use manually configured QEMU VM in a remote mode this could guide them through.
+
+This writing will focus on Windows, because there is no official QEMU support provided, but it should be relatively easy to adapt them for macOS use.
+
+## Prerequisites
+
+* Windows 11 or 10 with unix domain sockets support
+* SSH client feature installed on the machine
+* Hyper-V acceleration should be operational on the machine
+* Direcroty `C:\qemu-remote\` will be used for storing needed assets
+* Port `57561` free to use it for ssh over loopback interface
+
+## Obtaining and installing
+
+### QEMU
+
+Download QEMU (7.2.0 minimal) from https://qemu.weilnetz.de/w64/
+
+Then download the FCOS image for QEMU from https://getfedora.org/coreos/download?tab=metal_virtualized&stream=testing&arch=x86_64
+
+One will need `.xz` format extraction tool like xz itself or 7-zip. Use it to extract the `.qcow2` image to C:\qemu-remote\fedora-coreos-37.20221127.2.0-qemu.x86_64.qcow2
+
+With xz the command line (when run from the same directory) will be `xz -d fedora-coreos-37.20221127.2.0-qemu.x86_64.qcow2.xz`
```

Comment:
suggestion for consideration
```suggestion
With xz the command line (when run from the same directory) will be 
```
xz -d fedora-coreos-37.20221127.2.0-qemu.x86_64.qcow2.xz
```